livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java

```diff
@@ -131,6 +132,12 @@ public Integer createSession(InterpreterContext context, String kind) throws Exc
     }
   }
 
+  protected void initializeSpark(final InterpreterContext context,
+                                 final Map<String, Integer> userSessionMap) throws Exception {
+    interpret("val sqlContext = new org.apache.spark.sql.SQLContext(sc)\n" +
+        "import sqlContext.implicits._", context, userSessionMap);
```

Comment:
sqlContext is not available in livy 0.2 but available for later version. So here I will create sqlContext is it is not available.